src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -251,6 +252,7 @@ namespace
             float           m_turbidity;                // atmosphere turbidity
             float           m_radiance_multiplier;      // emitted radiance multiplier
             float           m_size_multiplier;          // sun size multiplier
+            float           m_distance;                 // distance between sun and earth
```

Comment:
In English, Sun is a proper noun so it's capitalized (and so would be Earth). Also, I would suggest using "scene" instead of "Earth" since the entire point of this exercise is to render scenes that take place on other places than the Earth :)